docs/scenarios_authoring/AggregatesInTimeWindows.md

```diff
@@ -65,27 +65,41 @@ Let’s map the above statement on the parameters of the Nussknacker Aggregate c
 
-> Historically Nussknacker used term *keyBy* instead of *groupBy*. Some of the screen shots used in the documentation may still use label *keyBy*.
-
 **aggregateBy** - this is an input to the aggregator; for each event  with the same groupBy value which qualiffies to the time window, the aggregateBy expression will be evaluated, fed to the aggregator and the aggregate will be updated.
 
-| groupBy | aggregateBy  | aggregator | result*  |
-|-------|--------------|------------|----------------------------------------------------------|
-|`#input.subscriberId` |`#input.value`| Sum | <p>`6000.0` for subscriberId = 1 </p> `200.0` for subscriberId = 2 |
-|`#input.subscriberId` |`1L`| Sum | <p>`3` for subscriberId = 1 </p> `1` for subscriberId = 2 |
-|`#input.subscriberId` |`{“tid”: #input.transactionId, “val”: #input.value}`| List |<p>`{{“tid”:11, “val”: 500.0},{“tid”:13, “val”: 5000.0},{“tid”:14, “val”: 1000.0}}` for subscriberId = 1 </p> `{{“tid”:12, “val”: 2000.0}}` for subscriberId = 2 |
+| groupBy               | aggregateBy                                          | aggregator   | result*                                                                                                                                                          |
+|-----------            | --------------                                       | ------------ | ----------------------------------------------------------                                                                                                       |
+| `#input.subscriberId` | `#input.value`                                       | Sum          | <p>`6000.0` for subscriberId = 1 </p> `200.0` for subscriberId = 2                                                                                               |
+| `#input.subscriberId` | `1L`                                                 | Sum          | <p>`3` for subscriberId = 1 </p> `1` for subscriberId = 2                                                                                                        |
+| `#input.subscriberId` | `{“tid”: #input.transactionId, “val”: #input.value}` | List         | <p>`{{“tid”:11, “val”: 500.0},{“tid”:13, “val”: 5000.0},{“tid”:14, “val”: 1000.0}}` for subscriberId = 1 </p> `{{“tid”:12, “val”: 2000.0}}` for subscriberId = 2 |
+
+
+*result is held in the variable configured in the `output` field.
 
-\*result is held in the variable configured in the `output` field.
+## Common behaviour
+Components which produce aggregates in time windows process multiple events; a question may arise about the contents of the variables when the window is closed.
+Presence of `#input` and `#inputMeta` variables depends on configuration of the `emitWhenEventLeft` parameter. `#key` parameter is added for every aggregation type and holds a key which is used in `groupBy` 
```

Comment:
Presence of variables defined before aggregation (e.g. `#input`, `#inputWithMeta`)